addons/account/static/src/components/bills_upload/bills_upload.js

```diff
@@ -128,15 +128,27 @@ AccountMoveUploadListRenderer.components = {
     AccountDropZone,
 };
 
-export class AccountMoveUploadListController extends ListController {}
-AccountMoveUploadListController.components = {
+export class AccountMoveListController extends ListController {
+    setup() {
+        super.setup();
+        this.account_move_service = useService("account_move");
+    }
+
+    async onDeleteSelectedRecords() {
+        if (!await this.account_move_service.addDeletionDialog(this, this.model.root.selection.map((r) => r.resId))) {
```

Comment:
Actually this doesn't work if you select all records
![image](https://github.com/odoo/odoo/assets/44770049/c2024299-0cc9-4fe6-a811-a7e496a83a45)

From a quick search I think you can use `getResIds`

Reply:
I could use `getSelectedResIds`